src/app/pages/my-virtrolio/my-virtrolio.component.ts

```diff
@@ -74,6 +93,41 @@ export class MyVirtrolioComponent implements OnInit {
     }
   }
 
+  shareLink() {
+    if (this.canShare()) {
+      this.navigator.share({
+        title: this.displayName + '\'s virtrolio!',
+        text: 'Visit this link to sign their virtrolio and send them a custom message!',
+        url: this.link,
+      })
+        .catch((error) => AuthService.displayError('Error sharing' + error));
+    } else {
+      AuthService.displayError('Share not supported on this device: ' + navigator.userAgent + 'Use a mobile device!');
```

Comment:
You can't say "use a mobile device" since this isn't supposed to happen in the first place.
```suggestion
      AuthService.displayError('Sharing Error: Native Share not supported on this browser: ' + navigator.userAgent);
```